docs/advanced_topics/amp.rst

```diff
@@ -1,5 +1,5 @@
-Building a site with AMP support
-================================
+How to Build a site with AMP support
```

Comment:
```suggestion
How to build a site with AMP support
```

I think we should keep this all lowercase like other pages.

Reply:
Okay. I am redoing it now.